core/controllers/reader.py

```diff
@@ -300,7 +300,121 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Method that initializes member variables for the handler.
+
+        Attributes:
+            exp_issues: ExplorationIssues. The exploration issues domain object.
+            issue_schema_version: int. The issue schema version.
+        """
+        super(StorePlaythroughHandler, self).__init__(*args, **kwargs)
+        self.exp_issues = None
+        self.issue_schema_version = None
+
+    def _find_playthrough_in_exp_issues(self, playthrough):
+        """Finds the given playthrough in the unresolved issues list of the
```

Comment:
I think this should be reworded to say that it's looking for an "equivalent" playthrough, not the exact playthrough itself (which implies that an "identical" playthrough is present).

To describe what I mean in code,

```python
lsts = [
    [1, 2],
    [1, 3],
]

for l in lsts:
    # What your doc "sounds" like it's doing, but this is always False.
    print([1,2] is l)
    # What the code is actually doing, and is True for index 0.
    print([1,2] == l)
```

Does that make sense?

Reply:
That is true. Done.